tests/test_logging.py

```diff
@@ -139,3 +141,197 @@ def test_request_data_is_not_saved_when_saving_disabled(self, m):
         requests.get("http://example.com/some-path?version=2.0")
 
         self.assertFalse(OutgoingRequestsLog.objects.exists())
+
+    @override_settings(LOG_OUTGOING_REQUESTS_DB_SAVE=True)
+    @override_settings(LOG_OUTGOING_REQUESTS_SAVE_BODY=True)
+    def test_logging_disallowed_content_type_request(self, m):
+        methods = [
+            ("GET", requests.get, m.get),
+            ("POST", requests.post, m.post),
+            ("PUT", requests.put, m.put),
+            ("PATCH", requests.patch, m.patch),
+            ("DELETE", requests.delete, m.delete),
+            ("HEAD", requests.head, m.head),
+        ]
+
+        for method, func, mocked in methods:
+            with self.subTest():
+                mocked(
```

Comment:
Can you please use `pytest` for the tests instead of the `unittest.TestCase` style?

You can then leverage `pytest.mark.parametrize` and fixtures to write nicer tests.

Additionally, `requests.foo` is just a wrapper around `requests.request("FOO", ...)`, so you only really need to define the array of methods and can use the underlying `requests.request(method, ...)` and `m.register_uri(method, ...)` calls (`m.get` is just as shortcut for `m.register_uri("GET", ...)`)